### PR TITLE
v4: Mention GPU acceleration fix in docs callout

### LINKED ISSUE #22610: [v4] Popovers looked blurred.
BS v4 popovers looked so indistinct in my browser.

It happened because the css property code `translateZ(0px)`.

I'm not sure whether it will influence , which I has opened **system DPI** on win10.
My browser is Vivaldi 1.9.818.49.

### PR DESCRIPTION
**Pull request by mdo**
Closes #22610

Preview: <https://deploy-preview-30811--twbs-bootstrap.netlify.app/docs/4.5/components/popovers/#usage>

### CODE REVIEW

**Comment by ffoodd** [comment]
Why not include the workaround right here?

**Comment by mdo** [comment]
> Why not include the workaround right here?

Personally I hadn't tested it and wasn't sure if we'd want to include some JS snippet here. I'm down to do that though if folks would prefer it.

**Comment by XhmikosR** [comment]
The snippet in https://github.com/twbs/bootstrap/issues/22610#issuecomment-373954635 assumes a bundler AFAICT. We need a snippet to cover all cases.

**Comment by mdo** [comment]
If there's some suggested code or specific change to make here we can get it in, otherwise we have to punt this to another v4.x release.

**Comment by Johann-S** [comment]
This snippet cover all cases @XhmikosR because if you import Popper.js with a `<script>` tag you'll be able to use Popper global variable.

so we can change it, to just that line:

```js
Popper.Defaults.modifiers.computeStyle.gpuAcceleration = !(window.devicePixelRatio < 1.5 && /Win/.test(navigator.platform));

**Comment by XhmikosR** [comment]
Cool, let's move the snippet to the docs and not link to an issue.

File: site/docs/4.5/components/popovers.md [hunk 1]
```diff
@@ -136,6 +136,13 @@ Enable popovers via JavaScript:
 
 {% highlight js %}$('#example').popover(options){% endhighlight %}
 
+{% capture callout %}
+##### GPU acceleration
+
+Popovers sometimes appear blurry in Windows 10 devices due to GPU acceleration and a modified system DPI. The workaround for this in v4 is to disable GPU acceleration as needed on your popovers. [See this issue for details and a suggested fix](https://github.com/twbs/bootstrap/issues/22610).
```
Comment (vanillajonathan) [inline site/docs/4.5/components/popovers.md:142]: "in Windows 10" or "on Windows 10 devices", but not "in Windows 10 devices".

**Comment by XhmikosR** [comment]
@Johann-S can you push your proposed snippet to this branch please?

**Comment by Johann-S** [comment]
done @XhmikosR 👍

**Review by XhmikosR — APPROVED** [review]

**Comment by KostblLb** [comment]
Still reproduced on 4.6,
i also use  @ng-bootstrap/ng-bootstrap@7.0.0, if it matters

**Pull request merged**